projects/architecture/interfaces/block-storage.proto

```diff
@@ -1,10 +1,8 @@
 syntax = "proto3";
 
-import "primitives/heartbeat.proto";
```

Comment:
These are all changes that were already merged to `master`. Did you rebase properly?

Reply:
it's the same PR, dunno why this change appears again